plugin/pkg/scheduler/algorithm/priorities/util/non_zero.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Can't we just move it to:
plugin/pkg/scheduler/algorithm/priorities/util.go
?
Do you need to create a package for it?